### DISCUSSION TITLE
No 'live' output for an action's step until the step completes

### DISCUSSION THREAD

**Discussion by jasonbelt** [community/community#67903] [Actions]
### Select Topic Area

Bug

### Body

You used to be able to see the live console output for a running github action step via the website.  Now however the output for a step is only viewable once the step completes.  For example, I opened the running workflow below and wasn't able to view ``Cache OSATE`` output until it completed and I won't be able to see ``Test``'s output until it completes.  Being able to monitor the output of step was useful as e.g. you might realize you pushed a regression and therefore will be able to resolve the issue sooner.

![issue](https://github.com/community/community/assets/2582573/3e733585-098c-43a7-a183-99c22b6691ad)

**Comment by topaxi** [comment]
I also started to notice that there is no more live output, this is especially bad for long running tasks where it's hard to gather at what step or command something is stuck at.

**Comment by Michael-Coveware** [comment]
I've also noticed this functionality stopped working recently on Chrome. My co-worker stated the **opposite** for Firefox - it didn't work before, and recently _started_ working.

**Comment by diofeher** [comment]
I'm seeing the same here.

**Comment by miberecz** [comment]
I'm struggling with the same issue, which is very inconvenient when running long jobs like Ansible or Terraform

**Comment by abbaty48** [comment]
Still the same problem here

**Comment by mara004** [comment]
Having the same issue today with some of pypdfium2's workflows.

**Comment by jonasjuffinger** [comment]
Having the same issue with custom actions. This would be really useful for long running actions.

**Comment by T1k-T1k** [comment]
Still the same problem here

**Reply by mara004** [reply]
Indeed, the issue seems to have resurfaced over the last couple of days

**Reply by amiraminbranex** [reply]
yes same here, resurfaced again

**Comment by burithetech** [comment]
+1 after yesterday issue with Actions there the issue again.
